### PR TITLE
Updated sensor device types according to specs

### PR DESCRIPTION
**Pull request by jadhavrohit924**
#### Problem
What is being fixed?  Examples:
* Sensor device types was not as per specs.

#### Change overview
Updated sensor device types according to specs

#### Testing
How was this tested? (at least one bullet point required)
* Verified that the xml is in sync with the specification.

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 1]
```diff
@@ -1242,23 +1242,12 @@ limitations under the License.
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
+            <include cluster="Binding" client="false" server="false" clientLocked="true" serverLocked="true">
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1245]: Why we we listing it at all, then?

Please create one PR per device type so they can be reviewed separately....

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 1 above)
Comment (woody-apple) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1245]: I think it's OK to have a mixed PR here, however, are the changes correct?

**Review by woody-apple — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 1 above)
Comment (bzbarsky-apple) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1245]: They are not correct, at first glance, and in the past when trying to fix up these sorts of changes we run into issues with some things being fixed and others being broken at the same time, leading to a lot of back-and-forth and failure to converge on correct changes.

Breaking up the PR is a suggestion to mitigate that....

I can try to work with the un-broken-up thing, but I can't commit to more than two rounds of review if we do it that way.

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 2]
```diff
@@ -1242,23 +1242,12 @@ limitations under the License.
...
-                <requireCommand>GetGroupMembershipResponse</requireCommand>
-                <requireCommand>RemoveGroup</requireCommand>
-                <requireCommand>RemoveGroupResponse</requireCommand>
-                <requireCommand>RemoveAllGroups</requireCommand>
-                <requireCommand>AddGroupIfIdentifying</requireCommand>
-            </include>
-            <include cluster="Boolean State" client="true" server="false" clientLocked="false" serverLocked="true"></include>
+	    <include cluster="Boolean State" client="false" server="true" clientLocked="true" serverLocked="true">
+		<requireAttribute>STATE_VALUE</requireAttribute>
+	    </include>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1250]: Please fix the indent here.

**Review by bzbarsky-apple — APPROVED** [review]
With the Binding thing for contact sensor removed and the indent fixed, this looks good.

**Comment by jadhavrohit924** [comment]
Will add per device type separate PR to avoid confusion and easy to review. Closing ths.

**Pull request closed without merging**